products_service/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "preinstall": "npm i nodemon -g",
+    "preinstall": "if [ \"$USER\" = root ]; then npm i nodemon -g; fi",
```

Comment:
Will this change work on all platforms?

Reply:
ah, it will only work on the platforms whose shell is `sh` and its derivatives. :(

The reason I made this change is because I updated the `Dockerfile`'s to run as the `node` user instead of `root`, and therefore doesn't have the permission to install npm packages globally ... 

ok, how about this: let's revert the change here, and make `npm install` in the `Dockerfile`'s skip running the script with `--ignore-scripts`. Will that work?

Reply:
Have you tried to run this without the preinstall script? Are we able to run `nodemon` from the devDependencies? If  so, I think we'll be set.

Reply:
@BrettHoyer I've updated the PR to use `nodemon` from the `devDependencies`, please take a look.